src/modules/DressMe/Sizes.tsx

```diff
@@ -0,0 +1,73 @@
+import styled from "styled-components";
+import { Field, Formik, useFormik } from "formik";
+import { validate } from "./validate";
+
+const Sizes = () => {
```

Comment:
rename to something more informative
SizesSelectors
OutfitSizeSelect
OutfitSizeSelection
OutfitSizeSection